LayoutTests/ipc/webpageproxy-correctionpanel-no-crash-expected.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
I would tend to do like the other tests and add something like
`Test passes if it does not crash.` in the html and which would show up there.